x/chainmanager/common_test.go

```diff
@@ -0,0 +1,22 @@
+package chainmanager_test
```

Comment:
How about moving this to test_helper.go (similar to what you did in Checkpoint module)?

Reply:
Okay